docs/api.rst

```diff
@@ -13,19 +92,16 @@ API documentation
 
 .. autoclass:: pytket.extensions.qiskit.AerBackend
     :special-members: __init__
-    :show-inheritance:
```

Comment:
These backends inherit from the internal class `_AerBaseBackend` which probably doesn't need to be shown in the docs.